Sources/snowman/main.swift

```diff
@@ -2,11 +2,17 @@ import SnowManLogic
 import SnowManDictionary
 
 let dictionaryManager = DictionaryManager(words: SnowManDictionary.words) 
-let guessManager = GuessManager(totalIncorrectGuessesAllowed: 10)
 let word = dictionaryManager.getRandomWord()
 
 let io = IO()
+
 let view = View(io: io)
+
+let maxNumberOfGuesses = 10
+view.promptNumberOfGuesses()
+let numberOfGuesses = view.receiveNumberOfGuesses(maxNumberOfGuesses)
```

Comment:
And that is why I moved it outside the game into the main lol. I interpreted game setup as in it should be done before the `Game` instance is created. I will try something and see if it flies, I'll tag you when I do.